### PR TITLE
fix/[ARTS-910] Role service local config

### PR DESCRIPTION
**Pull request by KateSant**
Local config to enable a developer to run role-service locally without a config server or dependency on the mts-trial-deployment-config repo

### Checklist:

- [x] Branch name follows convention (feature/arts-#-short-name OR fix/arts-#-short-name)
- [x] I have included a short-meaningful title, description and changes for this PR

### CODE REVIEW

**Review by darrengsmithson — CHANGES_REQUESTED** [review]
We seem to have lost application-local.yml - I think Tamir may have removed it - I think I added a comment to his review...anyhow, if application-local.yml is in the resources folder bootstrap will pick up the liquibase properties.

**Review by mdp0 — DISMISSED** [review]

**Comment by KateSant** [comment]
> We seem to have lost application-local.yml - I think Tamir may have removed it - I think I added a comment to his review...anyhow, if application-local.yml is in the resources folder bootstrap will pick up the liquibase properties.

Thanks Darren, do you mean this config should go in an application-local.yml rather than bootstrap-local.yml?

File: role-service/src/main/resources/bootstrap-local.yml [hunk 1]
```diff
@@ -6,3 +6,11 @@ spring:
         enabled: false
     discovery:
       enabled: false
+  liquibase:
```
Comment (tamirkamara) [inline role-service/src/main/resources/bootstrap-local.yml:9]: Can all these be moved to app local file?

Comment (KateSant) [reply role-service/src/main/resources/bootstrap-local.yml]: Done

**Review by darrengsmithson — APPROVED** [review]

**Review by mdp0 — APPROVED** [review]

**Pull request merged**